src/snowflake/cli/plugins/object/stage/diff.py

```diff
@@ -42,6 +42,47 @@ def has_changes(self) -> bool:
             or len(self.only_on_stage) > 0
         )
 
+    def __str__(self) -> str:
+        """
+        Method override for the standard behavior of string representation for this class.
+        """
+        components: List[
+            str
+        ] = (
+            []
+        )  # py3.8 does not support subscriptions for builtin list, hence using List
+
+        # The specific order of conditionals is for an aesthetically pleasing output and ease of readability.
```

Comment:
+ also why I did not do a simple `if-else` for each list attribute, because interleaving "There are no new files ..." with "here are the files..." is hard to read.